Upgrading to TS 4.5 results in `PostMessageOptions` not found

`typescript@4.5.2`, `@vueuse/core@6.9.1`

![image](https://user-images.githubusercontent.com/7195563/142388158-4785be24-5361-423b-bd04-8f80f7893bc8.png)

```node
node_modules/@vueuse/core/index.d.ts:2983:34 - error TS2304: Cannot find name 'PostMessageOptions'.

2983         (message: any, options?: PostMessageOptions | undefined): void;
                                      ~~~~~~~~~~~~~~~~~~


Found 1 error.
```

--- a/packages/core/useWebWorker/index.ts
+++ b/packages/core/useWebWorker/index.ts
@@ -4,6 +4,13 @@ import { ref, Ref, shallowRef } from 'vue-demi'
 import { tryOnScopeDispose } from '@vueuse/shared'
 import { ConfigurableWindow, defaultWindow } from '../_configurable'
 
+export interface UseWebWorkerReturn<Data = any> {
+  data: Ref<Data>
+  post: typeof Worker.prototype['postMessage']
+  terminate: () => void
+  worker: Ref<Worker | undefined>
+}
+
 /**
  * Simple Web Workers registration and communication.
  *
@@ -12,11 +19,11 @@ import { ConfigurableWindow, defaultWindow } from '../_configurable'
  * @param workerOptions
  * @param options
  */
-export function useWebWorker(
+export function useWebWorker<Data = any>(
   url: string,
   workerOptions?: WorkerOptions,
   options: ConfigurableWindow = {},
-) {
+): UseWebWorkerReturn<Data> {
   const {
     window = defaultWindow,
   } = options
@@ -59,5 +66,3 @@ export function useWebWorker(
     worker,
   }
 }
-
-export type UseWebWorkerReturn = ReturnType<typeof useWebWorker>

